templates/tasks.cron.erb

```diff
@@ -0,0 +1,7 @@
+SHELL=/bin/sh
+RAILS_ENV=production
+FOREMAN_HOME=/usr/share/foreman
+
+# Clean up expired tasks from the database
+<% cronline = scope.lookupvar("::foreman::plugin::tasks::cron_line") %>
```

Comment:
cronline is in the local scope so ```@cronline``` also works.

Reply:
Chmm, the tests don't seem to like this change